tvm_vm/src/executor/crypto.rs

```diff
@@ -169,7 +172,13 @@ fn check_signature(engine: &mut Engine, name: &'static str, hash: bool) -> Statu
         engine.modifiers.chksig_always_succeed || pub_key.verify(&data, &signature).is_ok();
     #[cfg(not(feature = "signature_no_check"))]
     let result = pub_key.verify(&data, &signature).is_ok();
+
+    let duration = start.elapsed();
+
+    println!("Time elapsed by chcksign is: {:?}", duration);
```

Comment:
Using println especially in libs is a bad practice, pls remove it

Reply:
fixed